website/docs/developer/deployment-guide/recovery-workflow.mdx

```diff
@@ -14,18 +14,18 @@ on which components can be modified and which components should not
 be modified. Refer to the legend on the diagram for details on which
 components can be modified.
 
-Recovery processes are kicked off manually by an operator through
-the Cumulus Dashboard. The dashboard calls an API which kicks off a
-recovery workflow. Recovery is an asynchronous operation since data
-requested from GLACIER can take up to 4 hours or more to reconstitute
-in most scenarios, and DEEP_ARCHIVE can take 12 hours. 
+A developer kicks off the recovery processes manually via the UI,
+which then triggers an API that kicks off a recovery workflow. Recovery is an asynchronous operation since data
+requested from GLACIER can take 4 hours or more to reconstitute,
+and DEEP_ARCHIVE can take 12 hours. 
 Since it is asynchronous, the recovery container
 relies on a database to maintain the status of the request and event
 driven triggers to restore the data once it has been reconstituted
 from archive into an S3 bucket. Currently data is copied back to the
-Cumulus S3 primary data bucket as the final restore step. Determining
-the status of the recovery job is done manually by querying the database
-directly or by checking the status on the Cumulus Dashboard.
+Cumulus S3 primary data bucket which is the default bucket. The developer 
+has the option to override the default bucket with another restore bucket if desired. 
+Determining the status of the recovery job is done manually by querying the database
```

Comment:
May want to add here that you can also check by calling the ORCA API. We do have a lambda that tells the status of a job.

Reply:
added. Yes we do